internal/server/recon/recognize_places.go

```diff
@@ -29,3 +30,97 @@ func RecognizePlaces(
 	// TODO(ws): Implement.
 	return nil, nil
 }
+
+func tokenize(query string) []string {
+	tokens := []string{}
+
+	// Split by space.
+	for _, partBySpace := range strings.Split(query, " ") {
```

Comment:
nit: just curious, does golang not allow iteration by char?

Reply:
Yes it can iterate by char -- just feels using 'split' is closer to how human thinks about tokenize things, and maybe slightly more readable?